### PR TITLE
Add documentation for default metrics exported by Istio 

### PR DESCRIPTION
**Pull request by gargnupur**

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit a3e55039fa38ba9d4b300ba743fb03b3466d3a39

https://app.netlify.com/sites/preliminary-istio/deploys/5b1adf1a0733d5034e5a481e

**Comment by gargnupur** [comment]
Previous deploy/netlify failed because of timeout connecting to an external site.. reran and it's passing now :)

File: content/docs/reference/config/policy-and-telemetry/metrics.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+title: Metrics
```
[the remark this answers is not in the document]
Comment (gargnupur) [reply content/docs/reference/config/policy-and-telemetry/metrics.md]: Fixed

File: content/docs/reference/config/policy-and-telemetry/metrics.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+description: Default Metrics exported from Istio through Mixer.
+weight: 50
+---
+
+Istio exports metrics through Mixer. They can be found
+[here](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml)
```
[the remark this answers is not in the document]
Comment (gargnupur) [reply content/docs/reference/config/policy-and-telemetry/metrics.md]: Fixed

File: content/docs/reference/config/policy-and-telemetry/metrics.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+---
+title: Metrics
+description: Default Metrics exported from Istio through Mixer.
+weight: 50
+---
+
+Istio exports metrics through Mixer. They can be found
```
[the remark this answers is not in the document]
Comment (gargnupur) [reply content/docs/reference/config/policy-and-telemetry/metrics.md]: Fixed

File: content/docs/reference/config/policy-and-telemetry/metrics.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+[here](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml)
+under the section with “kind: metric”. It uses [metric
+template](/docs/reference/config/policy-and-telemetry/templates/metric/) to define these metrics.
+
+We will describe metrics first and then the labels for each metric.
+
+## Metrics
+
+*   **Request Count**: This is a `COUNTER`
+    [metric](https://github.com/istio/istio/blob/b6fa713dc8356cb49bbc1bda37f2fd9b5bce1e31/install/kubernetes/templates/istio-mixer.yaml.tmpl#L786:9)
```
[the remark this answers is not in the document]
Comment (gargnupur) [reply content/docs/reference/config/policy-and-telemetry/metrics.md]: Thanks..Fixed

File: content/docs/reference/config/policy-and-telemetry/metrics.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+*   **Connection mTLS**: This identifies the service authentication policy of
+    the request. It is set to true, when istio is used to make secure
+    communications.
+
+    ```yaml
+       #Default Attribute Expression
+       connection_mtls: connection.mtls | false
+    ```
+
+## Example
```
[the remark this answers is not in the document]
Comment (gargnupur) [reply content/docs/reference/config/policy-and-telemetry/metrics.md]: Done

File: content/docs/reference/config/policy-and-telemetry/metrics.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+Request Count metric instance would look as follows:
+
+```yaml
```
[the remark this answers is not in the document]
Comment (gargnupur) [reply content/docs/reference/config/policy-and-telemetry/metrics.md]: Removed

**Comment by douglas-reid** [comment]
looks like netlify issues again. perhaps timeouts? ideas?

File: content/docs/reference/config/policy-and-telemetry/metrics.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+## Metrics
+
+*   **Request Count**: This is a `COUNTER`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L786:9)
+    incremented for every request handled by an Istio proxy.
+
+*   **Request Duration**: This is a `DISTRIBUTION`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L802:9)
+    which measures the duration of the request (as observed by the server-side
+    proxy). This metric is obtained from envoy proxy.
```
Comment (douglas-reid) [inline content/docs/reference/config/policy-and-telemetry/metrics.md]: Looking at this again, I think we should just drop `This metric is obtained from envoy proxy` from all the descriptions. I think this is more confusing than necessary, as envoy has its own metrics for clusters that differ from these metrics.

Comment (gargnupur) [reply content/docs/reference/config/policy-and-telemetry/metrics.md]: Thanks..Fixed

**Comment by gargnupur** [comment]
@douglas-reid : yes netlify was failing because of timeout previously.. it's passing now..

File: content/docs/reference/config/policy-and-telemetry/metrics.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    TCP connection as measured by the server-side proxy.
+
+## Labels
+
+*   **Source Service**: This identifies the source service responsible for an
+    incoming request. This is also the FQDN for a source service. Ex:
+    "reviews.default.svc.cluster.local".
+
+    ```yaml
+       #Default Attribute Expression
```
Comment (geeknoid) [inline content/docs/reference/config/policy-and-telemetry/metrics.md]: For this and the following 3 code blocks:

- Remove the Default Attribute Expression comment line, it's not helpful
- Remove the 3 character indent.

Comment (gargnupur) [reply content/docs/reference/config/policy-and-telemetry/metrics.md]: Thanks..FIxed

**Pull request merged**